js/main.js

```diff
@@ -35,11 +35,13 @@ var options =
   end_animation: 'fadeOut',       // default animate CSS for everything to be removed with at once
   alternateInterval: 3000,        // default time in ms between alternating time and interval
   disableAlternation: false,      // whether alternation should be disabled
+  showMessages: 'yes',            // whether to show messages - 'yes', 'no', or 'only' for no departures
```

Comment:
Truth be told, it doesn't need to be `'yes'`. It could be anything that isn't `'no'` or `'only'` (including not defined).